llvm.spec

```diff
@@ -78,6 +78,8 @@ rm -f %{i}/bin/FileRadar.scpt %{i}/bin/GetRadarVersion.scpt
 
 # Avoid dependency on /usr/bin/python, Darwin + Xcode specific
 rm -f %{i}/bin/set-xcode-analyzer
+# Remove all static libs except libomptarget-nvptx.a
+find %{i}/lib64 -name "*.a" -and -not -name "libomptarget-*.a" -delete
```

Comment:
@mrodozov , Instead of this you need to remove https://github.com/cms-sw/cmsdist/blob/bdfcb03effcafcd041143d9681899dd0f6fa5a42/llvm.spec#L19 line

Reply:
ah no, you want to keep `libomptarget-*.a` , ok ignore my previous comment

Reply:
as you want to keep `libomptarget-*.a` so your original change was correct.

Reply:
@mrodozov , FYI, please read the above comments

Reply:
yes I just figured it from the issue. so yes we go back one commit